docs/quickstart.rst

```diff
@@ -13,6 +13,10 @@ Parsl is available on PyPI, but first make sure you have Python3.5+
 
 Parsl has been tested on Linux and MacOS.
 
+.. note:: As of Parsl v0.7.2 we are switching to an opt-in model for anonymous usage tracking. We request that our
+   users volunteer to send us this information by setting ``PARSL_TRACKING=true`` in your environment or by
```

Comment:
grammar is weird here - "our users" is third person, "your" is 2nd person.

Reply:
replaced your with their